packages/tools-iottwinmaker/src/lib/policy/workspace-dashboard-role-policy.ts

```diff
@@ -0,0 +1,28 @@
+export const workspaceDashboardRolePolicyTemplate = {
```

Comment:
To confirm: does this match the latest roles we generate in the console? I think the one I recently generated there also had SiteWise and Lambda permissions?

Reply:
Yes, it depends on which permissions you select in the console, I'll update it to the "include all" instead of "no video permissions" as that seems more robust and can be changed later if needed?

Reply:
(can discuss offline) Sorry, can you clarify what you mean by "include all" ? Security will want us to ensure the permissions are least-privilege